app/src/test/java/org/simple/clinic/summary/linkId/LinkIdWithPatientViewLogicTest.kt

```diff
@@ -37,6 +38,8 @@ class LinkIdWithPatientViewLogicTest {
   private val uiEvents = PublishSubject.create<UiEvent>()
 
   private val patientUuid = UUID.fromString("755bfa1a-afa5-4c80-9ec7-57d81dff2ca1")
+  val patientName = "TestName"
```

Comment:
Not important to fix, but these don't need to be public since test classes are not used elsewhere.